subseas_flex_fcst/maproom.py

```diff
@@ -93,6 +93,28 @@ def display_relevant_control(variable):
         style_threshold=displayed_style
     return style_percentile, style_threshold
 
+@APP.callback(
+    Output("leadTime","options"),
+    Output("leadTime","value"),
+    Input("startDate","value"),
+)
+def targetStartOptions(startDate):
+    leadsDic = {}
+    for x in range(1,CONFIG["numberLeads"]+1):
+        dict = {}
+        dict["provider"] = CONFIG[f"dataLead{x}"]
```

Comment:
Here also the dictionary index names could be improved; currently `provider` is the provider's lead time rep, while `numVal` is the numerical value we use for calculations

Reply:
Aaron provided an example on how to write the config file. This should bear on this code so I'll review after you had a chance to revisit it.